### PR TITLE
Adding week 13 blog post

### PR DESCRIPTION
**Pull request by xtanion**
visible in `blog.html`

### CODE REVIEW

File: docs/source/posts/2022/2022-09-15-week-13-blog.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="1280" height="600" src="https://user-images.githubusercontent.com/74976752/190474528-9d66651b-032f-4c7d-9bb6-5ad140017d0c.mp4" frameborder="0"></iframe>
+
+
+What is coming up next week?
+----------------------------
+
+- Fixing the camera issue (As of writing this blog, it's done already)
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-09-15-week-13-blog.rst:30]: Not sure, but if its already done you can remove this.

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion ,
Nice Blog and that preview looks amazing 🤩.

**Comment by m-agour** [comment]
Hello @xtanion,
Animation looks so cool! And post has no issues.

**Comment by skoudoro** [comment]
Thank you @xtanion, looks good, merging

**Pull request merged**